src/platforms/javascript/common/sourcemaps/uploading/typescript.mdx

```diff
@@ -3,7 +3,7 @@ title: TypeScript (tsc)
 description: 'Upload your source maps using tsc and Sentry CLI.'
 sidebar_order: 2
 notSupported:
-  - javascript.svelte
+  - javascript.sveltekit
```

Comment:
Was it a mistake that svelte was marked not supported before? It's not clear why it should be removed here